packages/11ty/_plugins/iiif/helpers/get-image-service.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path')
+/**
+ * Return path to a figure entry's `info.json`
+ * 
+ * @param  {Object} iiifConfig
+ * @param  {Object} figure
+ * @return {String}            Path to figure's `info.json`
+ */
+module.exports = (eleventyConfig, figure) => {
+  const { imageServiceDirectory, outputDir } = eleventyConfig.globalData.iiifConfig
+  const { src } = figure
+  if (!src) return
+  const { name } = path.parse(src)
+  return src.startsWith('http')
+    ? src
+    : path.join('/', outputDir, name, imageServiceDirectory, 'info.json')
```

Comment:
This all looks good to merge and makes sense to me -- I do wonder if this ^ thing we do in like 4-5 different places where we check if a string starts with `http` and conditionally render a either the original `src` or some different relative path is worth abstracting into some kind of utility (separate task)

Reply:
Wrote up a ticket fer it: https://jira.getty.edu/browse/DEV-12941